profiler/profiler_test.go

```diff
@@ -59,20 +61,30 @@ func TestStart(t *testing.T) {
 	})
 
 	t.Run("options/GoodAPIKey/Agent", func(t *testing.T) {
+		rl := &log.RecordLogger{}
```

Comment:
There isn't really a need for a pointer. This could well be
```suggestion
		var rl log.RecordLogger
```